content/docs/1.1.0/what-is-longhorn.md

```diff
@@ -44,4 +47,4 @@ Common front-ends include a Linux kernel device (mapped under /dev/longhorn) and
 
 ### Specify Schedules for Recurring Snapshot and Backup Operations
 
-Specify the frequency of these operations (hourly, daily, weekly, monthly, and yearly), the exact time at which these operations are performed (e.g., 3:00am every Sunday), and how many recurring snapshots and backup sets are kept.
\ No newline at end of file
+Specify the frequency of these operations (hourly, daily, weekly, monthly, and yearly), the exact time at which these operations are performed (e.g., 3:00am every Sunday), and how many recurring snapshots and backup sets are kept.
```

Comment:
?

Reply:
I think this is caused my vim format. When I do 'git diff' it shows me
```
-Specify the frequency of these operations (hourly, daily, weekly, monthly, and yearly), the exact time at which these operations are performed (e.g., 3:00am every Sunday), and how many recurring snapshots and backup sets are kept.
\ No newline at end of file
+Specify the frequency of these operations (hourly, daily, weekly, monthly, and yearly), the exact time at which these operations are performed (e.g., 3:00am every Sunday), and how many recurring snapshots and backup sets are kept.
```